rust/src/pgsql/pgsql.rs

```diff
@@ -119,6 +118,7 @@ pub enum PgsqlStateProgress {
     CommandCompletedReceived,
     ErrorMessageReceived,
     ConnectionTerminated,
+    #[cfg(test)]
```

Comment:
Not sure I understand this here...

Reply:
`#cfg` in rust applies only to the next one, so just `UnknownState`